# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/base/content/test/performance/browser_startup_syncIPC.js
# Commit: 786823305560
# Full Hash: 786823305560b03316cb758207f76838388603af
# Author: sotaro <sotaro.ikeda.g@gmail.com>
# Date: 2020-05-05 03:49:49
# Regressor Bug: 1574746
# File Overlap Count: 1
# Description:
#   Bug 1574746 - Remove AllowWebRenderForThisWindow() r=nical
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D71440
# ==============================================================================

diff -r 7af7b4435997 -r 786823305560 browser/base/content/test/performance/browser_startup_syncIPC.js
--- a/browser/base/content/test/performance/browser_startup_syncIPC.js	Mon May 04 23:13:55 2020 +0000
+++ b/browser/base/content/test/performance/browser_startup_syncIPC.js	Sun May 03 11:38:53 2020 +0000
@@ -191,6 +191,11 @@
       ignoreIfUnused: true, // intermittently occurs in "before becoming idle"
       maxCount: 1,
     },
+    {
+      name: "PWebRenderBridge::Msg_EnsureConnected",
+      condition: WIN && WEBRENDER,
+      maxCount: 1,
+    },
   ],
 
   // Things that are expected to be completely out of the startup path